### DISCUSSION TITLE
OpenAI response object

### DISCUSSION THREAD

**Discussion by Kulja** [vanna-ai/vanna#429] [Q&A]
Is there any way to get the actual response object we get from OpenAI (as opposed to only the relevant message with generated SQL)? I am trying to show how many tokens are used in the API call and I can see the information is there, I just can't get to it.

**Comment by zainhoda** [comment]
Which setup are you referring to? Using OpenAI directly?

**Reply by Kulja** [reply]
And apparently, I don't know how to post a proper reply

**Comment by Kulja** [comment]
this configuration https://vanna.ai/docs/bigquery-openai-standard-chromadb/

**Reply by Kulja** [reply]
And when I say that I see it, I mean that I know it is in this response object https://github.com/vanna-ai/vanna/blob/main/src/vanna/openai/openai_chat.py#L123
